devicetypes/smartthings/zigbee-motion-detector.src/zigbee-motion-detector.groovy

```diff
@@ -171,7 +172,7 @@ def configure() {
 	def manufacturer = getDataValue("manufacturer")
 	if (manufacturer == "eWeLink") {
 		sendEvent(name: "checkInterval", value:2 * 60 * 60 + 5 * 60, displayed: false, data: [protocol: "zigbee", hubHardwareId: device.hub.hardwareID, offlinePingable: "1"])
-		return zigbee.configureReporting(zigbee.POWER_CONFIGURATION_CLUSTER, 0x0021, DataType.UINT8, 30, 3600, 0x10) + refresh()
+		return zigbee.configureReporting(zigbee.POWER_CONFIGURATION_CLUSTER, 0x0021, DataType.UINT8, 3600, 7200, 0x10) + refresh()
```

Comment:
So You want to change  power configuration reporting also for the  already supported eWelink MS01 ?

Reply:
Yes, keep the same configuration.